recipes/susi/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "susi" %}
+{% set version = "1.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/susi-{{ version }}.zip
+  sha256: 88f1958c1177da69e2d64fadc5855076c5d20e536f53d1b957eadaaf7735c70c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - joblib >=0.13.0
+    - numpy >=1.18.5
+    - python >=3.6
+    - scikit-learn >=0.21.1
+    - scipy >=1.3.1
+    - tqdm >=4.45.0
+    - matplotlib >=3.3.0
```

For details see https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib

```suggestion
    - matplotlib-base >=3.3.0
```